From 5e95788bf9d547f2e80d853c65fd45c246403b52 Mon Sep 17 00:00:00 2001
From: Radha Mohan Chintakuntla <rchintakuntla@cavium.com>
Date: Fri, 9 Nov 2018 11:46:16 -0800
Subject: [PATCH 173/767] misc: otx_bphy_ctr: Increase the maximum BPHY
 interrupts to be supported

commit 2d9d2ed567ca56399763f35ec0ac3c2ce071eca3 from
git@git.assembla.com:cavium/WindRiver.linux.git

This patch increases the maximum BPHY interrupts that can be supported.

Change-Id: I46308420fa28ef809af8b44f022ae7747ac14867
Signed-off-by: Radha Mohan Chintakuntla <rchintakuntla@cavium.com>
Reviewed-on: https://sj1git1.cavium.com/8029
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Tested-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/misc/otx_bphy_ctr.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/misc/otx_bphy_ctr.c b/drivers/misc/otx_bphy_ctr.c
index dceff42c8e67..05087050a2e6 100644
--- a/drivers/misc/otx_bphy_ctr.c
+++ b/drivers/misc/otx_bphy_ctr.c
@@ -20,7 +20,7 @@
 
 #define DEVICE_NAME	"otx-bphy-ctr"
 #define OTX_IOC_MAGIC	0xF3
-#define MAX_IRQ		4
+#define MAX_IRQ		23
 
 static struct device *otx_device;
 static struct class *otx_class;
-- 
2.31.1

